examples/hybrid_search.py

```diff
@@ -0,0 +1,122 @@
+import turbopuffer as tpuf
+
+
+# Helper to convert the results to a dictionary with ranks
+def results_to_ranks(results, reverse=False):
+    if reverse:
+        results = sorted(results, key=lambda item: item.dist, reverse=True)
+    return {item.id: rank for rank, item in enumerate(results, start=1)}
+
+
+# Fuses two search result sets together into one
+# Uses reciprocal rank fusion
+def rank_fusion(bm25_results, vector_results, k=60):
+    # Compute the ranks of all docs
+    # Note: a higher score in bm25 denotes a better result, whereas a higher score
+    # with vector euclidean distance indicates a worse result. When computing RRF,
+    # we reverse the order of vector results so that higher = better.
+    bm25_ranks = results_to_ranks(bm25_results)
+    vector_ranks = results_to_ranks(vector_results, reverse=True)
+
+    print("bm25_ranks:", bm25_ranks)
+    print("vector ranks", vector_ranks)
+
+    # Get all the unique document IDs
+    doc_ids = set(bm25_ranks.keys()).union(set(vector_ranks.keys()))
+
+    # Calculate the RRF score for each document
+    scores = {}
+    for doc_id in doc_ids:
+        score = 0.0
+        if doc_id in bm25_ranks:
+            score += 1.0 / (k + bm25_ranks[doc_id])
+        if doc_id in vector_ranks:
+            score += 1.0 / (k + vector_ranks[doc_id])
```

Comment:
Since the scores here can be quite different, don't you need to [normalize it](https://medium.com/plain-simple-software/distribution-based-score-fusion-dbsf-a-new-approach-to-vector-search-ranking-f87c37488b18) if you're not going by the position alone?